engines/wintermute/configure.engine

```diff
@@ -1,3 +1,3 @@
 # This file is included from the main "configure" script
 # add_engine [name] [desc] [build-by-default] [subengines] [base games] [deps]
-add_engine wintermute "Wintermute" yes "" "" "jpeg png zlib vorbis 16bit highres"
```

Comment:
This has some interesting side-effects that I'd have to handle before merging this. Namely that as far as I know, Dreamcat would be the _only_ game that works with ScummVM, if built without any of the above requirements.